counterparty-core/counterpartycore/lib/check.py

```diff
@@ -959,9 +959,14 @@ def software_version():
             config.PROTOCOL_CHANGES_URL, headers={"cache-control": "no-cache"}, timeout=10
         )
         versions = json.loads(response.text)
-    except (requests.exceptions.ConnectionError, ConnectionRefusedError, ValueError) as e:  # noqa: F841
+    except (
+        requests.exceptions.ConnectionError,
+        ConnectionRefusedError,
+        ValueError,
+        TimeoutError,
+    ) as e:  # noqa: F841
         logger.warning("Unable to check version! " + str(sys.exc_info()[1]))
```

Comment:
Is this what we want to log?

Reply:
nope! I think `logger.error("Unable to check version:" + str(e)`

Reply:
yeah I think `error`